GNUmakefile

```diff
@@ -46,5 +46,14 @@ drone:
 	drone lint .drone/drone.yml
 	drone sign --save grafana/terraform-provider-grafana .drone/drone.yml
 
+golangci-lint:
+	docker run \
+		--rm \
+		--interactive \
+		--tty \
```

Comment:
Are these needed for this?

Reply:
@inkel - Good call; I don't believe those are necessary and I've removed them. Thanks!